docs/csharp/language-reference/compiler-messages/iterator-yield.md

```diff
@@ -82,7 +79,6 @@ The body of an iterator method must conform to restrictions on the `yield return
 - **CS1626**: *Cannot yield a value in the body of a try block with a catch clause*
 - **CS1631**: *Cannot yield a value in the body of a catch clause*
 - **CS1629**: *Unsafe code may not appear in iterators*
-- **CS9237**: *''yield return' should not be used in the body of a lock statement*
```

Comment:
Can you remove line 89 as well?  That line states that you can't use `yield return` in a `lock` statement block.